helm/mongodb/templates/stateful-set.yaml

```diff
@@ -22,9 +22,12 @@ spec:
         - containerPort: 27017
         - containerPort: 28017
         livenessProbe:
-          tcpSocket:
-            port: 27017
           initialDelaySeconds: 15
+          exec:
+            command:
+            - /bin/bash
+            - -c
+            - ping -c 1 mongodb-0.mongodb && ping -c 1 mongodb-1.mongodb && ping -c 1 mongodb-2.mongodb
```

Comment:
Can we also combine with a telnet on 27017 to keep the initial behavior ?

Reply:
Telnet for the local mongo or all of them? They get started up sequentially so the latter option is not possible.